ces_admin/management/commands/hash_identities.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+import hashlib
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from rapidsms.models import Connection
+from decisiontree.models import Session
+
+class Command(BaseCommand):
+    help = 'Hashes the user identities, i.e., phone numbers'
+
+    def handle(self, *args, **options):
+        five_minutes_ago = datetime.datetime.now() - datetime.timedelta(minutes=5)
+        one_day_ago = datetime.datetime.now() - datetime.timedelta(days=1)
+        # Find connections from sessions that do not have a state_id, i.e., the user completed the survey or canceled their session
+        # Add a five minute boundary, in the rare event that Rapidsms or Twilio 
```

Comment:
seems like part of this sentence has been cut off?

Reply:
Why, yes. I've improved the comment overall....